DiffusionPreprocessing/scripts/eddy_postproc.sh

```diff
@@ -114,7 +114,7 @@ else # Combining across diffusion directions with opposing phase-encoding polari
 
 	# Average Eddy-Rotated bvecs. Get for each direction the two b matrices, average those and then eigendecompose the average b-matrix to get the new bvec and bval.
 	# Also outputs an index file (1-based) with the indices of the input (Pos/Neg) volumes that have been retained in the output
-	${globalscriptsdir}/average_bvecs.py ${eddydir}/Pos.bval ${eddydir}/Pos_rotated.bvec ${eddydir}/Neg.bval ${eddydir}/Neg_rotated.bvec ${datadir}/avg_data ${eddydir}/Pos_SeriesVolNum.txt ${eddydir}/Neg_SeriesVolNum.txt
+	${globalscriptsdir}/average_bvecs.py ${eddydir}/Pos.bval ${eddydir}/Pos_rotated.bvec ${eddydir}/Neg.bval ${eddydir}/Neg_rotated.bvec ${datadir}/avg_data ${CombineDataFlag} ${eddydir}/Pos_SeriesVolNum.txt ${eddydir}/Neg_SeriesVolNum.txt
```

Comment:
Have you tested that this works?  Because based on the definition of `main()` in `average_bvecs.py`, it doesn't appear to me that we've properly provided inputs for `bvalsoutfile`, `bvecsoutfile`, and `indicesoutfile`.

Reply:
Yeah, I've tested running this. The usage for the updated average_bvecs.py is:
`usage: average_bvecs bvals1 bvecs1 bvals2 bvecs2 output_basename combine_data_flag [overlap1 overlap2]`

This matches the way I run the script. `bvalsoutfile`, `bvecsoutfile`, and `indicesoutfiles` are defined based on the `output_basename`